backend/tests/model_factories.py

```diff
@@ -55,6 +55,16 @@ def dict_factory(factory, **kwargs):
         return partial(dict_factory, cls)()
 
 
+class UserFactory(GenerateDataMixin, factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+
+    username = factory.LazyFunction(fakers.fake_username)
+    password = factory.Faker("password")
+    is_staff = True
```

Comment:
No, this needs to come from UserFactory.create(is_admin=True).

Please, remove this line.

Reply:
are you sure? it doesn't work.

Reply:
https://docs.djangoproject.com/en/5.2/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser

It's `is_superuse`, my bad.